src/Umbraco.Cms.ManagementApi/Filters/EnumOutputFormatter.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.AspNetCore.Mvc.Formatters;
+
+namespace Umbraco.Cms.ManagementApi.Filters;
+
+public class EnumOutputFormatter : SystemTextJsonOutputFormatter
```

Comment:
Suggest renaming to `ManagementApiJsonOutputFormatter` - we may end up doing more here than enum formatting.